frontend/src/store/modules/place/actions.js

```diff
@@ -44,40 +44,38 @@ export default {
         });
     },
 
-    likePlace: (context, placeId) => {
-        return new Promise((resolve, reject) => {
-            httpService.post('places/' + placeId + '/like')
-                .then(function (res) {
-                    getLikedPlace(context, placeId);
-                    resolve(res);
-                })
-                .catch(function (err) {
-                    reject(err);
-                });
-        });
-    },
-    
-    dislikePlace: (context, placeId) => {
-        return new Promise((resolve, reject) => {
-            httpService.post('places/' + placeId + '/dislike')
-                .then(function (res) {
-                    getLikedPlace(context, placeId);
-                    resolve(res);
-                })
-                .catch(function (err) {
-                    reject(err);
-                });
-        });
-    },
-
     getLikedPlace: (context, placeId) => {
         httpService.get('places/' + placeId + '/liked')
-            .then(function (res) {
+            .then( (res) => {
                 const likeStatus = [ STATUS_LIKED, STATUS_DISLIKED, STATUS_NONE ]
-                    .indexOf(res.data.data) === -1 ? STATUS_NONE : res.data.data;
+                    .indexOf(res.data.data.liked) === -1 ? STATUS_NONE : res.data.data.liked;
```

Comment:
No need to .indexOf, because api already returns one of [STATUS_LIKED, STATUS_DISLIKED, STATUS_NONE] values